### DISCUSSION TITLE
Multiple Language support

### DISCUSSION THREAD

**Discussion by arch-fan** [aliencaocao/netease_cloudmusic_discord_rpc#22] [Ideas]
Yeah, it would be interesting to support different languages, I can translate it to English and Spanish if needed

**Comment by aliencaocao** [comment]
What do you mean? Which part of this program are you referring to? I don't see any place that needs to be translated

**Reply by arch-fan** [reply]
I mean multiple language support for the RPC strings, eg "Listen on Netease" button

**Reply by aliencaocao** [reply]
Sure, you can make a PR. Right now it is already in english, I can take care of simplified chinese

**Reply by arch-fan** [reply]
Ok, gimme some days because I don't have much free time

**Reply by arch-fan** [reply]
The question is, how do you want to give multi language support? Different branches, config file...?

**Reply by aliencaocao** [reply]
It seems like unlike discord bots, discord RPC dont support multi language yet. I guess we cant do this for now until discord add this feature
